src/main/java/clevernotbot/ToDo.java

```diff
@@ -0,0 +1,57 @@
+package clevernotbot;
+
+/**
+ * Represents a more specific task with no time limit attached to it.
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructor for ToDo.
+     *
+     * @param name      Name of the task.
+     * @param completed The status of the task. (Completed or not)
+     */
+    public ToDo(String name, boolean completed) {
+        super(name, completed);
+    }
+
+    /**
+     * Returns the deadline for Todo which is just infinite.
+     *
+     * @return Deadline for Todo.
+     */
+    @Override
+    public String getTime() {
+        return "inf";
+    }
+
+    /**
+     * Returns The type of task.
+     *
+     * @return Type of task.
+     */
+    @Override
+    public String getTaskType() {
+        return "T";
+    }
+
+    /**
+     * Toggles the completion of todo task.
+     *
+     * @return A toggled version of todo. (Completed = true -> Completed = false)
+     */
```

Comment:
You can omit header comments for overriding methods (provided the parent method's Javadoc applies exactly as is to the overridden method).